action.yml

```diff
@@ -4,35 +4,27 @@ inputs:
   NO_COVERAGE:
     description: 'set to a non-empty string to suppress gathering coverage'
     required: false
-    type: string
     default: ''
   GAP_TESTFILE:
     description: 'Name of the GAP file to be read for executing the package tests (overrides TestFile in PackageInfo.g)'
     required: false
-    type: string
     default: ''
   only-needed:
-    description: 'If set to true then only needed dependencies of the package being tested are loaded'
+    description: 'If set to a non-empty string, then only needed dependencies of the package being tested are loaded'
     required: false
-    type: boolean
-    default: false
+    default: ''
   load-all:
-    description: 'If set to true then execute LoadAllPackages() before the package being tested'
+    description: 'If set to a non-empty string, then execute LoadAllPackages() before the package being tested'
     required: false
-    type: boolean
-    default: false
+    default: ''
   warnings-as-errors:
-    description: 'If set to true then any errors produced whilst loading the package will be treated as errors'
+    description: 'If set to a non-empty string, then any errors produced whilst loading the package will be treated as errors'
     required: false
-    type: boolean
-    default: true
+    default: 'true'
   pre-gap:
     description: 'Prefix for the ''GAP'' shell variable used by this action to launch GAP (e.g. setting this to ''valgrind --trace-children=yes --leak-check=full'' will run GAP through valgrind)'
     required: false
-    type: string
     default: ''
-env:
-  CHERE_INVOKING: 1
```

Comment:
Since this is not a valid way of specifying an environment variable in the workflow, `CHERE_INVOKING` is never set (hence why I removed it). If we *do* want to set the environment variable, we can do that as explained in the schema:

> If you want to modify the environment variable stored in the workflow, use `echo "{name}={value}" >> $GITHUB_ENV` in a composite step.